src/registrering/kategori-spørsmål/KategoriSpørsmål.tsx

```diff
@@ -33,6 +35,7 @@ const KategoriSpørsmål: FunctionComponent<Props> = ({
     return (
         <section className="kategori-spørsmål">
             <Undertittel className="blokk-xxs">{tittel}</Undertittel>
+            <Normaltekst className="kategori-spørsmål__kategorihjelpetekst">{hjelpetekst}</Normaltekst>
```

Comment:
`hjelpetekst` er satt som optional, hva skjer om vi ikke sender med noen `hjelpetekst` her? Vises da `null` eller `undefined` vises på skjermen?

Reply:
Det vises ingenting i de tilfellene. Men jeg kan legge til en sjekk rundt for å ta bort elementet fra domen når det ikke er hjelpetekst.

Reply:
Da blir ikke margin bottom jeg la til med når det ikke er hjelpetekst. Jeg antar at det er riktig at den ikke skal være med.